yarn-project/aztec.js/src/fee/private_fee_payment_method.ts

```diff
@@ -28,10 +27,15 @@ export class PrivateFeePaymentMethod implements FeePaymentMethod {
     private wallet: Wallet,
 
     /**
-     * A secret to shield the rebate amount from the FPC.
-     * Use this to claim the shielded amount to private balance
+     * Address that the FPC sends notes it receives to.
      */
-    private rebateSecret = Fr.random(),
+    private feeRecipient: AztecAddress,
```

Comment:
This is code is from PrivateRefundPaymentMethod which was in refunds test before.